R/tm_g_swimlane.R

```diff
@@ -36,7 +36,7 @@
 #'
 #' @examples
 #' # Example using stream (ADaM) dataset
-#' data <- teal.data::cdisc_data() |>
+#' data <- cdisc_data() |>
 #'   within(library(dplyr)) |>
 #'   within(library(nestcolor)) |>
 #'   within(ADSL <- osprey::rADSL %>%
```

Comment:
Since `osprey` is a `Depends`, the prefix can be dropped.

Reply:
thanks for spotting this. removed prefixes in the most recent commit